app/models/status_message.rb

```diff
@@ -128,7 +128,11 @@ def update_and_dispatch_attached_photos(sender)
   end
 
   def comment_email_subject
-    message.title length: 70
+    if (self.public?)
+      message.title length: 70
+    else
+      I18n.t('status_messages.limited_subject')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.